src/main/scala/com/codacy/client/bitbucket/v2/service/BuildStatusServices.scala

```diff
@@ -12,7 +14,9 @@ class BuildStatusServices(client: BitbucketClient) {
    *
    */
   def getBuildStatus(owner: String, repository: String, commit: String, key: String): RequestResponse[BuildStatus] = {
-    val url = s"https://bitbucket.org/api/2.0/repositories/$owner/$repository/commit/$commit/statuses/build/$key"
+    val commitBuildStatusesUrl = generateCommitBuildStatusesUrl(owner, repository, commit)
+    val encodedKey = URLEncoder.encode(key, "UTF-8")
```

Comment:
Codacy found an issue: [The string literal 'UTF-8' appears 5 times in the file.](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5907044)